tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -68,12 +68,15 @@ def start_node(self, node, timeout_sec=60, **kwargs):
 
         node.account.mkdirs(OpenMessagingBenchmarkWorkers.PERSISTENT_ROOT)
 
-        start_cmd = f"cd /opt/openmessaging-benchmark; \
-                      HEAP_OPTS=\" \" \
-                      bin/benchmark-worker \
-                      --port {OpenMessagingBenchmarkWorkers.PORT} \
-                      --stats-port {OpenMessagingBenchmarkWorkers.STATS_PORT} \
-                      >> {OpenMessagingBenchmarkWorkers.STDOUT_STDERR_CAPTURE} 2>&1 & disown"
+        start_cmd = (
+            f"cd /opt/openmessaging-benchmark; "
```

Comment:
also changed this to use this other apparently more pythonic pattern for multi-line strings, because all the intendation space was actually ending up in the command (hard to read in the logfile).